index.html

```diff
@@ -731,9 +731,9 @@ <h3>
           Extensions to the <code>Window</code> object
         </h3>
         <p>
-          The following extensions to the <dfn data-cite=
-          "!HTML#window"><code>Window</code></dfn> object specify the <a>event
-          handler attributes</a> on which events relating to the
+          The following extensions to the <code><dfn data-cite=
+          "HTML/window-object.html#window">Window</dfn></code> object specify
```

Comment:
I'd be quite surprised if this was correct... you usually don't need to specify the exact html page, right? Just `document#id`.

What error message is this solving?

Reply:
The link checker can't follow the links, because the links are actually handled with JS in the HTML spec. So, this avoid the JS redirect in the HTML spec, allowing the link checker to work correctly. 

Admittedly, we are working around a limitation in link checker 🤔

Reply:
OK, it seems annoying to have to remember to do this everywhere but if that's what we have to do...

Reply:
We can sit on most of these for now... there are some other actual broken ones that can be seen by following the link in #735.